app/views/products/show.html.erb

```diff
@@ -30,36 +30,10 @@
      data-markers="<%= @markers.to_json %>"
      data-mapbox-api-key="<%= ENV['MAPBOX_API_KEY'] %>"></div>
 
-<article class="container">
-  <section class="product-nutrition product-section">
-    <h4 class="sub-heading">NUTRITION</h4>
-    <table class='styled-table'>
-      <tr>
-        <th>Nutritional info</th>
-        <th>Per 100g</th>
-      </tr>
-      <tr>
-        <td>Calories</td>
-        <td><%= @product.calories_per_100g %></td>
-      </tr>
-      <tr>
-        <td>Carbohydrates</td>
-        <td><%= @product.carbs_per_100g %></td>
-      </tr>
-      <tr>
-        <td>Protein</td>
-        <td><%= @product.protein_per_100g %></td>
-      </tr>
-      <tr>
-        <td>Fat</td>
-        <td><%= @product.fat_per_100g %></td>
-      </tr>
-    </table>
-  </section>
-  <section class="product-ingredients product-section">
-    <h4 class="sub-heading">INGREDIENTS</h4>
-    <%= @product.ingredients %>
-  </section>
+
+
+ <article class="container">
+  <%= render 'products/tabs_show' %>
```

Comment:
can you rename tabs_show? reason being it sounds like it can be re-used as tabs elsewhere but it is one-purpose only in this case for nutrition and ingredients tabs

Reply:
Renamed to product_show_tabs